dockerfiles/KerasContainerDockerfile

```diff
@@ -0,0 +1,13 @@
+ARG REGISTRY
+ARG CODE_VERSION
+ARG RPC_VERSION
+FROM ${REGISTRY}/${RPC_VERSION}-rpc:${CODE_VERSION}
+
+RUN pip install -q keras
```

Comment:
How about set Keras' version like this, `keras==2.2.*`? See https://pypi.org/project/Keras/#history.